src/styles/card.css

```
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  width: 166px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.card__image-position {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__image {
+  max-width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  display: flex;
+  justify-content: start;
+  padding-top: 40px;
```

use margins for indents between elements, don't use paddings for that, fix it everywhere
```suggestion
  margin-top: 40px;
```